docs/content/docs/guide/running.md

```diff
@@ -38,17 +38,24 @@ run a PipelineRun on CI:
 - The author of the pull request has permissions to push to branches inside the
   repository.
 
-- The author who initiated the pull request is identified in an `OWNERS` file
+- The author who initiated the pull request is identified in an `OWNERS` files
   found in the main directory of the branch that is set as the default branch
   on GitHub or your other service provider.
 
-  The OWNERS file need adheres to a specific format, similar to the Prow OWNERS
-  file format (available at <https://www.kubernetes.dev/docs/guide/owners/>),
-  with the exception that we do not yet support the use of `OWNERS_ALIASES`.
+  The OWNERS file adheres to a specific format, similar to the Prow OWNERS
+  file format (available at <https://www.kubernetes.dev/docs/guide/owners/>). We
+  support simple OWNERS configuration including `approvers` and `reviewers` lists
+  and they are treated equally in terms of permissions for executing a PipelineRun.
+  If the OWNERS file includes `filters` instead of a simple OWNERS configuration,
```

Comment:
The original request to support OWNERS_ALIASES came from a project using this https://github.com/openshift/assisted-service/blob/master/OWNERS. They only use ".*" to manage the approvers/reviewers access and the rest of the filters to manage labels.
[The owners guide](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners) also mentions "WARNING: The approve plugin does not currently respect filters. Until that is fixed, filters should only be used for the labels key (as shown in the above example).". Now it looks like there is some progress on the issue linked and I can check this further, but for the time being, supporting ".*" is one step better than not supporting filters at all and that's exactly what the original requester need.

Re "default behavior", I do not think there is default behavior in this case. One can either use simple config (approvers:[], reviewers:[]) OR filters, but not mix them. So if they use filters, they have to use ".*" to achieve the behavior of the simple config.

Reply:
okay fine since you already implemented it :) but that's a bit weird imho for the people who dont really understand prow's owners files and its intricacies while reading the doc...